### PR TITLE
Update sample codes

### PR DESCRIPTION
**Pull request by JordanBeziau**
- update config to use dynamic data
- update samples to use config variables

The goal is to be able to setup examples with any merchant shop regardless of his whitelabel.

### CODE REVIEW

File: www/sample/config.php [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
+ **/
+
+// SUBSTITUTE THE KEY NUMBER BY YOUR MERCHANT SHOP (Menu: Settings > Shop > API REST Keys)
+ define('USERNAME', 'KEY Number 1');
+ define('PASSWORD', 'KEY Number 2');
+ define('PUBLIC_KEY', 'KEY Number 3');
+ define('SHA_KEY', 'KEY Number 4');
+ define('SERVER', 'KEY Number 5');
+ $URL_JS = 'KEY Number 6';
+
```
Comment (yyoyo) [inline www/sample/config.php:17]: You should add a "!!!CHANGEME!!!" visible here

File: www/sample/paid.php [hunk 2]
```diff
@@ -1,51 +1,101 @@
...
             <strong>Numéro de la commande :</strong>
+
             <?php echo $answer['kr-answer']['orderDetails']['orderId'] ; ?>
+
             <br />
+
          </div>
+
    </body>
+
```
Comment (yyoyo) [inline www/sample/paid.php:59]: Why did you add so many new lines ?

**Review by yyoyo — CHANGES_REQUESTED** [review]
See comments

**Comment by JordanBeziau** [comment]
@yyoyo added new commit with your feedbacks
